w10h01/test/pgdp/trials/TrialOfTheDreamsTest.java

```diff
@@ -0,0 +1,87 @@
+package pgdp.trials;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import java.util.function.Function;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class TrialOfTheDreamsTest {
+    private Function<byte[], Boolean> getLock() {
+        return (byte[] bytes) -> {
+            if (bytes.length != combination.length) {return false;}
+            for (int i = 0; i < bytes.length; i++) {
+                if (bytes[i] != combination[i]) {return false;}
+            }
+            return true; 
+        }
+    }
+
+    @Test
+    @DisplayName("Combination of Length 1")
+    void TestLengthOne() {
+        byte[] combination = {Byte.MAX_VALUE};
+        Function<byte[], Boolean> lock = getLock();
```

Comment:
```suggestion
        Function<byte[], Boolean> lock = getLock(combination);
```
You can provide combination with an argument.